clara/c_parser.py

```diff
@@ -525,6 +527,9 @@ def visit_scanf(self, node, args):
                                    line=node.coord.line)
             self.addexpr(VAR_IN,
                          Op('ListTail', Var(VAR_IN), line=node.coord.line))
+            vars_list.append(a)
```

Comment:
This cannot be right: `a` can also be an array assignment expression. In which case would this be necessary?

Reply:
It is a try to add support of `stdin` input to C parser-interpreter.
In C language, it's not possible to read an array directly from `stdin`, so `a` could not be an array. 🤔

Reply:
You can do `scanf("%d", array[i])` to read the `i`-th element of the `array`.

Reply:
So it's getting hard. :)) I'll take a look at code again.